pkg/sliceutils/map.go

```diff
@@ -51,3 +51,17 @@ func Map(slice, mapFunc interface{}) interface{} {
 	}
 	return outSlice.Interface()
 }
+
+// MapsIntersect returns true there is at least one key-value pair that is present in both maps
+// If both, or either maps are empty, it returns false
+// TODO : Implement this so that it can take map[interface{}]interface{}
```

Comment:
`map[interface{}]interface{}` won't help, since we almost never use them. This will be a good use case for generics though :)

Reply:
Starting in Go 1.18

Reply:
Updated comment